### PR TITLE
Measure class weights in LocalDataset

### PR DESCRIPTION
**Pull request by almazan**
This PR fixes a bug in the `measure_weights` functions in `LocalDataset`.

### CODE REVIEW

File: darwin/dataset/local_dataset.py [hunk 1]
```diff
@@ -275,3 +281,6 @@ def __str__(self):
             f"  Number of images: {len(self.images_path)}\n"
             f"  Number of classes: {len(self.classes)}"
         )
+
+    def __add__(self, dataset):
+        return self.extend(dataset, extend_classes=True)
```
Comment (andreaazzini) [inline darwin/dataset/local_dataset.py:286]: I'm not sure I like the idea of using `__add__`, which is less explicit than `self.extend(dataset, extend_classes=True)`

File: darwin/torch/dataset.py [hunk 2]
```diff
@@ -96,23 +95,35 @@ def __getitem__(self, index: int):
...
         if self.transform is not None:
             img = self.transform(img)
 
+        target = self.get_target(index)
+
+        return img, target
+
+    def get_target(self, index: int):
+        """Returns the classification target
+        """
```
Comment (andreaazzini) [inline darwin/torch/dataset.py:108]: Honestly not sure if a formatter would make this change. In any case, let's make sure we run the formatter!

```suggestion
        """
        Returns the classification target
        """
```

**Review by andreaazzini — CHANGES_REQUESTED** [review]
Just a question: do we need to override `LocalDataset.__add__`? Do we actually use that anywhere?

Otherwise, just make sure to run a formatter and I'll approve

File: darwin/dataset/local_dataset.py (hunk 1 above)
Comment (almazan) [reply darwin/dataset/local_dataset.py:286]: I think I used it in the auto-annotate code, but not anymore, so I can safely remove it.

**Review by andreaazzini — APPROVED** [review]

**Pull request merged**